sapi/fpm/tests/set-pm-max-spawn-rate.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+FPM: set pm.max_spawn_rate
+--SKIPIF--
+<?php
+include "skipif.inc";
+?>
+--FILE--
+<?php
+
+require_once "tester.inc";
+
+$cfg = <<<EOT
+[global]
+error_log = {{FILE:LOG}}
+log_level = notice
+[unconfined]
+listen = {{ADDR}}
+pm = dynamic
+pm.max_children = 5
+pm.start_servers = 2
+pm.min_spare_servers = 1
+pm.max_spare_servers = 3
+pm.max_spawn_rate = 64
+EOT;
+
+$tester = new FPM\Tester($cfg);
+$tester->start(['-t', '-t']);
+foreach ($tester->getLogLines(-1) as $line) {
+  if (strpos($line, "pm.max_spawn_rate") !== false) {
+      $configOption = explode("NOTICE:", $line);
+      echo trim($configOption[1]) . "\n";
+      break;
+  }
+}
```

Comment:
It would be great to move this to the tester as it seems like something that could be used elsewhere too. Maybe something like `expectLogConfigOptions($options)`, have `$options` as a list (key = configuration name, value = its value) and fail if one of the options do not match.